debian/control

```diff
@@ -21,7 +21,7 @@ Standards-Version: 3.9.8
 Vcs-Git: https://anonscm.debian.org/git/pkg-xorg/lib/libxfont.git
 Vcs-Browser: https://anonscm.debian.org/git/pkg-xorg/lib/libxfont.git
 
-Package: libxfont1
+Package: libxfont-1.5.2
```

Comment:
The current libxfont binary package is called `libxfont2`. I would adopt the naming here and use `libxfont1`.